packages/vsce/__tests__/__e2e__/specs/jvmEndpoint.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { expect, test } from "@playwright/test";
+import { constants, findAndClickTreeItem, getTreeItem, prepareZoweExplorerView, resetWiremock, resetZoweExplorerView } from "../utils/helpers";
+
+test.beforeEach(async ({ page, request }) => {
+  await resetWiremock(request);
+  await prepareZoweExplorerView(page);
+});
+
+test.afterEach(async ({ page }) => {
+  await resetZoweExplorerView(page);
+});
+
+test.describe("JVM Endpoint tests", () => {
+  test("should expand JVM servers tree to reveal JVM servers", async ({ page }) => {
+    await findAndClickTreeItem(page, constants.PROFILE_NAME);
+    await findAndClickTreeItem(page, constants.CICSPLEX_NAME);
+    await findAndClickTreeItem(page, constants.REGION_NAME);
+    await findAndClickTreeItem(page, "JVM Servers");
+
+    await expect(getTreeItem(page, constants.JVM_SERVER_1_NAME)).toBeVisible();
+    await expect(getTreeItem(page, constants.JVM_SERVER_1_NAME)).toHaveText(constants.JVM_SERVER_1_NAME);
+  });
```

Comment:
This is the same test as in JVM server tests - I don't think we need to do it twice.

If this functionality is not working, it will fail in the test below as it performs the same action, plus the jvm endpoint
```suggestion
```